packages-proprietary/tokens/src/components/ams/page-header.compact.tokens.json

```diff
@@ -4,6 +4,9 @@
       "brand-name": {
```

Comment:
Do we want to mark this as a breaking change? It technically isn't, but users will probably have to remove some padding from their mega menus.